server/src/Api.hs

```diff
@@ -23,11 +23,11 @@ api = Proxy
 
 -- types
 
-type ItemId = Integer
+type ItemId = Int
```

Comment:
I always got an error when using Integer. I assume that I did not correctly derived the type previously. Will have a closer look at that.